packages/craco-esbuild/src/index.js

```diff
@@ -12,15 +12,20 @@ module.exports = {
     context: { paths },
   }) => {
     const useTypeScript = fs.existsSync(paths.appTsConfig);
+    const esbuildLoaderOptions = pluginOptions.esbuildLoaderOptions;
+
+    // add includePaths custom option, for including files/components in other folders than src
+    // Used as in addition to paths.appSrc, optional parameter.
+    const optionalIncludes = esbuildLoaderOptions && esbuildLoaderOptions.includePaths || [];
```

Comment:
`includePaths` should probably be outside `esbuildLoaderOptions` as it is not passed down the esbuild loader. It's probably better to take it from `pluginOptions.includePaths`.

Reply:
Can you also udate the README to show how to use the option please? :)

Reply:
Added